app/views/layouts/one_column.html.erb

```diff
@@ -33,12 +33,20 @@
             <%= image_tag 'rubygamedev_logo.png', alt: 'RubyGameDev.com', width: 225, class: 'visible-xs', style: 'margin-top: 8px;' %>
           </a>
         </div>
+
         <div class="navbar-collapse collapse">
-          <ul class="nav navbar-nav">
-            <li><%= render 'layouts/language_dropdown' %></li>
-          </ul>
+          <ul class="nav navbar-nav navbar-right pull-right">
+
+             <li class="search"><%= form_tag(posts_path, :method => "get", id: "search-form") do %>
```

Comment:
put the start of `<%= form_tag...` onto a new line. It's best to match the indentation on opening and closing tags.